tools/migrate/to_supabase.js

```diff
@@ -19,6 +20,7 @@ const { SUPABASE_URL, SERVICE_ROLE_KEY } = process.env;
     url: feed,
     wiki_author_name: `${firstName} ${lastName}`,
     type: 'blog',
+    hash: hash(feed),
```

Comment:
We should normalize the feed url before hashing so that it's consistent with `feedId` in `Redis`
```js
const normalizeUrl = require('normalize-url');
hash: hash(normalizeUrl(feed))
```